packages/paste-core/layout/grid/__tests__/__snapshots__/grid.test.tsx.snap

```diff
@@ -78,8 +82,8 @@ exports[`Grid render should render responsive css 1`] = `
   -ms-flex-wrap: wrap;
   flex-wrap: wrap;
   display: flex;
-  margin-right: spaceNegative10;
-  margin-left: spaceNegative10;
+  margin-right: space0;
+  margin-left: space0;
```

Comment:
I can't immediately tell is this is because the test is different or whether it legitimately caught the negative margins being dropped from the outer Grid? 

This might be a bug

Reply:
Yeah it was a bug in the sense that these should have been defaulting to `auto` for the margin, not `space0`. However, this is a change related to the fix for the spacing not applied for vertical grids. In this case, the `vertical` value is an array of responsive values and for this test, the screen size is `vertical`. Thus, the right and left margins are cleared.